services/fee-estimator/tests/functional/feeEstimates.test.js

```diff
@@ -28,8 +28,32 @@ const lowTrafficMockup = require('../blockGenerator/lowTraffic.json');
 const moderateTrafficMockup = require('../blockGenerator/moderateTraffic.json');
 const highTrafficMockup = require('../blockGenerator/highTraffic.json');
 
+const transformTransactions = (block) => {
+	const { transactions } = block;
+
+	block.transactions = transactions.map(transaction => {
+		transaction.fee = BigInt(transaction.fee);
+		transaction.minFee = BigInt(transaction.size * 1000);
+		return transaction;
+	});
+};
+
+const transformBlocks = (traffic) => {
+	traffic.blocks.map(block => transformTransactions(block));
+};
```

Comment:
Can we please update the mockups instead of this? The updated mockups should match the current transactions response structure.